### PR TITLE
[opt]: Compatible with previous metaserver

### PR DESCRIPTION
**Pull request by 201341**
### What problem does this PR solve?

Issue Number: #2671  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
@wuhongsong PTAL

**Review by wuhongsong — APPROVED** [review]

**Comment by wuhongsong** [comment]
@Ziy1-Tan PTAL

File: curvefs/src/metaserver/metaserver.cpp [hunk 1]
```diff
@@ -136,8 +136,11 @@ void Metaserver::InitLocalFileSystem() {
 void InitS3Option(const std::shared_ptr<Configuration>& conf,
                   S3ClientAdaptorOption* s3Opt) {
     LOG_IF(FATAL, !conf->GetUInt64Value("s3.batchsize", &s3Opt->batchSize));
-    LOG_IF(FATAL, !conf->GetBoolValue("s3.enableBatchDelete",
-                                      &s3Opt->enableBatchDelete));
+    bool ret =
+        conf->GetBoolValue("s3.enableBatchDelete", &s3Opt->enableBatchDelete);
+    LOG_IF(WARNING, ret == false)
```
Comment (Ziy1-Tan) [inline curvefs/src/metaserver/metaserver.cpp:141]: whty not `LOG_IF(WARNING, !ret)`?

**Review by Ziy1-Tan — APPROVED** [review]

**Pull request merged**